.env.example

```
@@ -5,9 +5,100 @@ OLLAMA_BASE_URL='http://localhost:11434'
+
+# ===== EXAMPLE: IT DEPARTMENT SHAREPOINT CONFIGURATION =====
+# (Uncomment and configure when IT Department is onboarded)
+# IT_SHP_ID_APP='' # IT Azure AD application client ID
+# IT_SHP_ID_APP_SECRET='' # IT Azure AD application client secret
+# IT_SHP_TENANT_ID='' # IT Azure AD tenant ID  
+# IT_SHP_SITE_URL='' # URL of IT SharePoint site
+# IT_SHP_ORG_NAME='Information Technology (IT)' # IT organization name
+# IT_SHP_DOC_LIBRARY='IT' # IT document library path
+# IT_SHP_DEFAULT_SEARCH_FOLDERS='IT/Policies,IT/Documentation,IT/Projects' # IT search folders
+
+
```

dotenv-linter (4.0.0)

[warning] [ExtraBlankLine] Extra blank line detected

(ExtraBlankLine)